pkgs/applications/audio/soundconverter/default.nix

```diff
@@ -0,0 +1,55 @@
+{ pkgs, python3Packages, lib, fetchurl }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "soundconverter";
+  version = "4.0.3";
+
+  src = fetchurl {
+    url = "https://launchpad.net/soundconverter/trunk/${version}/+download/${pname}-${version}.tar.gz";
+    sha256 = "sha256-hzIG/4LD3705erPYvXb7uoRwF9LtKKIKB3jrhpYMsZ0=";
+  };
+
+  # FIXME: Tests still fail with:
+  # 'No GSettings schemas are installed on the system'
+  doCheck = false;
+
+  buildInputs = with pkgs; [
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-ugly
+    gtk3
+    python3Packages.distutils_extra
+    wrapGAppsHook
+  ];
+
+  nativeBuildInputs = with pkgs; [
+    intltool
+    gobject-introspection
+  ];
+
+  propagatedBuildInputs = with pkgs; [
+    gobject-introspection
```

Comment:
Build tools should go to `nativeBuildInputs`, see https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md.

Reply:
Sure.

Reply:
Though as I stated in https://github.com/NixOS/nixpkgs/pull/152733#issuecomment-1003086671 `gobject-introspection` has to be in both or it doesn't work.

Reply:
That might have just been caused by `strictDeps`. Setup hooks in general do not like it.

Reply:
`gobject-introspection` is still a build tool (or rather a setup hook).

Reply:
Again, being a build tools, it should go to `nativeBuildInputs`.